package.xml

```diff
@@ -17,47 +17,37 @@
   <url type="bugtracker">https://github.com/ros-planning/moveit2_tutorials/issues</url>
 
   <buildtool_depend>ament_cmake</buildtool_depend>
-  <build_depend>moveit_common</build_depend>
 
-  <build_depend>pluginlib</build_depend>
+  <depend>control_msgs</depend>
+  <depend>interactive_markers</depend>
+  <depend>moveit_core</depend>
+  <depend>moveit_ros_perception</depend>
+  <depend>moveit_ros_planning_interface</depend>
+  <depend>moveit_servo</depend>
+  <!-- <depend>moveit_visual_tools</depend> -->
+  <!-- <depend>pcl_conversions</depend> -->
+  <!-- <depend>pcl_ros</depend> -->
+  <depend>pluginlib</depend>
+  <!-- <depend>rosbag</depend> -->
+  <depend>rviz_visual_tools</depend>
+  <depend>tf2_eigen</depend>
+  <depend>tf2_geometry_msgs</depend>
+  <depend>tf2_ros</depend>
+
   <build_depend>eigen</build_depend>
-  <build_depend>moveit_core</build_depend>
-  <build_depend>moveit_ros_planning</build_depend>
-  <build_depend>moveit_ros_planning_interface</build_depend>
-  <build_depend>moveit_ros_perception</build_depend>
-  <build_depend>interactive_markers</build_depend>
   <build_depend>geometric_shapes</build_depend>
-  <!-- <build_depend>moveit_visual_tools</build_depend> -->
-  <build_depend>rviz_visual_tools</build_depend>
-  <!-- <build_depend>pcl_ros</build_depend> -->
-  <!-- <build_depend>pcl_conversions</build_depend> -->
-  <!-- <build_depend>rosbag</build_depend> -->
-  <build_depend>tf2_ros</build_depend>
-  <build_depend>tf2_eigen</build_depend>
-  <build_depend>tf2_geometry_msgs</build_depend>
+  <build_depend>moveit_common</build_depend>
+  <build_depend>moveit_ros_planning</build_depend>
 
-  <!-- <run_depend>franka_description</run_depend> -->
-  <run_depend>pluginlib</run_depend>
-  <run_depend>moveit_core</run_depend>
-  <!-- <run_depend>moveit_commander</run_depend> -->
-  <run_depend>moveit_fake_controller_manager</run_depend>
-  <run_depend>moveit_resources_panda_moveit_config</run_depend>
-  <run_depend>moveit_ros_planning_interface</run_depend>
-  <run_depend>moveit_ros_perception</run_depend>
-  <run_depend>interactive_markers</run_depend>
-  <!-- <run_depend>moveit_visual_tools</run_depend> -->
-  <run_depend>rviz_visual_tools</run_depend>
-  <run_depend>joint_state_publisher</run_depend>
-  <run_depend>robot_state_publisher</run_depend>
-  <run_depend>joy</run_depend>
-  <!-- <run_depend>pcl_ros</run_depend> -->
-  <!-- <run_depend>pcl_conversions</run_depend> -->
-  <!-- <run_depend>rosbag</run_depend> -->
-  <run_depend>rviz2</run_depend>
-  <run_depend>tf2_ros</run_depend>
-  <run_depend>tf2_eigen</run_depend>
-  <run_depend>tf2_geometry_msgs</run_depend>
-  <run_depend>xacro</run_depend>
+  <!-- <exec_depend>franka_description</exec_depend> -->
+  <exec_depend>joint_state_publisher</exec_depend>
+  <exec_depend>joy</exec_depend>
+  <!-- <exec_depend>moveit_commander</exec_depend> -->
+  <exec_depend>moveit_fake_controller_manager</exec_depend>
```

Comment:
This package needs to be removed or it will fail CI. Vatan just [applied](https://github.com/ros-planning/moveit2_tutorials/pull/90/files#diff-37a67ff78eb7260214b323353263b9af40a2fa98719a1e81937fae0159df87a3) this fix in #90

Reply:
Just rebased and removed that depend